src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -48,6 +50,18 @@ public CommandResult execute(Model model) throws CommandException {
         ArrayList<Appointment> patientAppointments = personToDelete.getAppointments();
         for (Appointment appointment : patientAppointments) {
             model.deleteAppointment(appointment);
+            Doctor targetDoctor = null;
+            for (Doctor doctor : doctorList) {
```

Comment:
Same thing. If possible, there might be a better way to write this with in built functions